### PR TITLE
Remove unused components

### PR DESCRIPTION
**Pull request by peterszerzo**
### Summary

Removing `Tile`, `Record`, `Timeline` and `Header` components. Moving snapshot tests in with the individual components. Fix blacklist bug on checking imports. 

### To be tested

Tester 1 (Imo)

- [x] Remaining components look and work as expected

Tester 2 (Tej)

- [x] Remaining components look and work as expected
- [x] Netlify looks good

### CODE REVIEW

**Comment by ImogenF** [comment]
Is component/docs/getting-started.md actually being used anywhere? If so, it uses the `Header` component so might be a good idea to take that out. Otherwise looks good!

**Comment by peterszerzo** [comment]
@ImogenF I'll deal with the imported `Header` component when it updates. Took out the docs, though.

File: packages/components/src/__tests__/libExports.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { readdirSync, readFileSync } from "fs"
 import { join } from "path"
 
-const BLACK_LIST = ["Internals", "utils", "__tests__", "types.ts", "index.ts"]
+const BLACK_LIST = ["Internals", "utils", "__tests__", "types.ts", "index.ts", ".DS_Store"]
```
Comment (fabien0102) [inline packages/components/src/__tests__/libExports.ts]: I'm surprise that nobody add this before ^^ 🍎 🍏

Comment (peterszerzo) [reply packages/components/src/__tests__/libExports.ts]: You were supposed to remember? Then you started ranting about how Macs shouldn't generate this file :)

**Pull request merged**